operator/pkg/cli/infer.go

```diff
@@ -562,12 +562,30 @@ func getDataSize(shape []int64) int64 {
 	return tot
 }
 
+func isNilOutputContents(contents *v2_dataplane.InferTensorContents) bool {
+	if contents == nil {
+		return true
+	} else {
+		if contents.Fp32Contents == nil &&
+			contents.Fp64Contents == nil &&
+			contents.BoolContents == nil &&
+			contents.UintContents == nil &&
+			contents.Uint64Contents == nil &&
+			contents.BytesContents == nil &&
+			contents.IntContents == nil &&
+			contents.Int64Contents == nil {
```

Comment:
:thought_balloon: Checking for just null values might not be very reliable.  We probably want to check that `len(contents.X) == 0`, as this will cover both the null case and the not-null-but-empty case.

Reply:
I think we might want to be careful for non-null empty as that might be a valid result, e.g. existence